extensions/panache/rest-data-panache/deployment/src/main/java/io/quarkus/rest/data/panache/deployment/properties/ResourcePropertiesProvider.java

```diff
@@ -19,6 +23,8 @@ public class ResourcePropertiesProvider {
     private static final DotName METHOD_PROPERTIES_ANNOTATION = DotName.createSimple(
             io.quarkus.rest.data.panache.MethodProperties.class.getName());
 
+    private static final List<String> ANNOTATIONS_TO_COPY = List.of("javax.annotation.security");
```

Comment:
Could you make it `RolesAllowed.class.getPackageName()`. I'd like to make sure it's rewritten by the Jakarta scripts.